providers/amazon/tests/unit/amazon/aws/triggers/test_mwaa.py

```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from unittest import mock
+from unittest.mock import AsyncMock
+
+import pytest
+
+from airflow.providers.amazon.aws.hooks.mwaa import MwaaHook
+from airflow.providers.amazon.aws.triggers.mwaa import MwaaDagRunCompletedTrigger
+from airflow.triggers.base import TriggerEvent
+from airflow.utils.state import DagRunState
+from unit.amazon.aws.utils.test_waiter import assert_expected_waiter_type
+
+BASE_TRIGGER_CLASSPATH = "airflow.providers.amazon.aws.triggers.mwaa."
+TRIGGER_KWARGS = {
+    "external_env_name": "test_env",
+    "external_dag_id": "test_dag",
+    "external_dag_run_id": "test_run_id",
+}
+
+
+class TestMwaaDagRunCompletedTrigger:
+    def test_init_states(self):
+        trigger = MwaaDagRunCompletedTrigger(**TRIGGER_KWARGS)
+        assert trigger.success_states == {DagRunState.SUCCESS.value}
+        assert trigger.failure_states == {DagRunState.FAILED.value}
+        acceptors = trigger.waiter_config_overrides["acceptors"]
+        expected_acceptors = [
+            {
+                "matcher": "path",
+                "argument": "RestApiResponse.state",
+                "expected": DagRunState.SUCCESS.value,
+                "state": "success",
+            },
+            {
+                "matcher": "path",
+                "argument": "RestApiResponse.state",
+                "expected": DagRunState.FAILED.value,
+                "state": "failure",
+            },
+            {
+                "matcher": "path",
+                "argument": "RestApiResponse.state",
+                "expected": DagRunState.RUNNING.value,
+                "state": "retry",
+            },
+            {
+                "matcher": "path",
+                "argument": "RestApiResponse.state",
+                "expected": DagRunState.QUEUED.value,
+                "state": "retry",
+            },
+        ]
+        assert len(acceptors) == len(DagRunState)
+        assert {tuple(sorted(a.items())) for a in acceptors} == {
+            tuple(sorted(a.items())) for a in expected_acceptors
+        }
```

Comment:
Could just be a mypy oddity, but the formatting here looks off to me.   If it passes static checks, then feel free to ignore this.

Reply:
Yes, I formatted it in a different way but pre-commit changed it to this

Reply:
Strange, alright.  Sorry about that.